lib/src/cli.dart

```diff
@@ -35,12 +35,13 @@ cli(
     @Rest(help: getTaskHelp, allowed: allowedTasks)
     List<String> tasks,
     {
-     @Flag(help: 'Print the version of grinder.')
+     @Flag(help: 'Print the version of grinder.', defaultsTo: false)
      bool version,
```

Comment:
sgtm. I like this better as well; I got a bit confused by the optional params and param annotations -